lib/console.js

```diff
@@ -54,6 +56,8 @@ function Console(stdout, stderr, ignoreErrors = true) {
   prop.value = createWriteErrorHandler(stderr);
   Object.defineProperty(this, '_stderrErrorHandler', prop);
 
+  this[kCounts] = {};
```

Comment:
I've updated to `Object.create(null)`... the symbol restriction is actually there to approximate the behavior implemented by browsers. We can easily support `Symbol` if we wanted to.